pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -257,6 +258,12 @@ func validateTaskTemplate(task v1alpha1.TaskSpec, job *v1alpha1.Job, index int)
 		if container.SecurityContext != nil && container.SecurityContext.Privileged != nil {
 			coreTemplateSpec.Spec.Containers[i].SecurityContext.Privileged = nil
 		}
+		if _, found := container.Resources.Limits["nvidia.com/gpu"]; found {
```

Comment:
please add a comment, assume this is for gpu share

Reply:
the comment is added.

Reply:
is there any const for "nvidia.com/gpu"? if not, let's add a new one.

Reply:
not any const for it now. 

What about other GPU types, like ascend, AMD?

Reply:
Add the ascend-310. Others are not supportted so far.